src/csharp/GeneratorParams.cs

```diff
@@ -51,6 +50,11 @@ public void SetInputSequences(Sequences sequences)
             Result.VerifySuccess(NativeMethods.OgaGeneratorParamsSetInputSequences(_generatorParamsHandle, sequences.Handle));
         }
 
+        public void AddExtraInput(string name, Tensor value)
```

Comment:
Same comment for naming as for C interface.

Reply:
Does C API expect a zero terminated string and does ToUtf8 append zero terminator?